src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$transition: 300ms;
```

Comment:
As i understand, it's not the best practice to set transition for all, slightly better to specify exactly what will be changing.

But I think it works fine in this context. Only, in this case consider adding the 'all' keyword to your variable.